src/python/setup.py

```diff
@@ -0,0 +1,7 @@
+from setuptools import setup, find_packages
+setup(
+    name="circllhist",
+    version="0.1",
+    install_requires=['cffi'],
+    packages=['circllhist'],
+)
```

Comment:
`generatePythonFiles.sh` should maybe be done as part of `setup.py`

Reply:
Could be. The only reason I use setup.py here, is b/c it's the only sane way to make sure we install into correct directory (respecting virtualenvs, etc.). In fact I call setup.py from make now.

Feel free to improve setup.py. It would be great if this could be published as self contained module on pypi. For my use case, this is not needed.